xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/variables.less

```diff
@@ -11,3 +11,11 @@
 @border-width:                   1px;
 @headings-font-weight:           400; // See: http://jira.xwiki.org/browse/XWIKI-10838
 @nav-tabs-active-link-hover-bg:  @xwiki-page-content-bg;
+
+//== Notifications Email colors
+//
+//##
+
+@notification-email-bg: #F5F5F5;
+@notification-email-inner-bg: #ffffff;
+@notification-email-text-color: #1d1f24;
```

Comment:
there is a top section, called "// New variables provided by XWiki". Could have been added there. Although that category might be added at the bottom, since it might expand faster.